src/com/starkindustries/project/StarkDatabase.java

```diff
@@ -353,31 +353,37 @@ public boolean editComment(int c_id, String description, Connection conn) throws
 		else return true;
 	}
 	
-	public boolean voteQuestion(Question q, String type, Connection conn) throws SQLException {
+	public boolean voteQuestion(int q_id, String type, Connection conn) throws SQLException {
```

Comment:
Could you revert voteQuestion to previous one? I have already connected the vote and your previous one is more straightforward imo.